source/agora/script/Script.d

```diff
@@ -61,6 +61,22 @@ public struct Script
         this.opcodes = opcodes;
     }
 
+    /***************************************************************************
+
+        Params:
+            opcodes = the set of assumed to be validated opcodes
+
+        Returns:
+            a Script with the given opcodes which are assumed to be valid
+
+    ***************************************************************************/
+
+    public static Script assumeValidated (const(ubyte)[] opcodes)
```

Comment:
It's kind of difficult to identify what the function is for by just reading the name, `assumeValidated`. Isn't it better to change the name?

Reply:
Updated to `fromPreValidated`

Reply:
I think the name is consistent with:
- https://dlang.org/library/std/range/assume_sorted.html
- https://dlang.org/library/std/string/assume_utf.html
- https://dlang.org/library/std/exception/assume_unique.html
- https://dlang.org/library/std/exception/assume_wont_throw.html

E.g. a function that "casts" between two types based on an assumption.

Reply:
Ah, there is a history behind it. :)

Reply:
ok will revert this rename.